ethexe/contracts/src/IRouter.sol

```diff
@@ -45,6 +48,11 @@ interface IRouter {
     /// @param blobTxHash The transaction hash that contains the WASM blob. Set to zero if applied to the current transaction.
     event CodeValidationRequested(bytes32 codeId, bytes32 blobTxHash);
 
+    /// @notice Emitted when validators for the next era are set.
```

Comment:
What'd you say if we make this event informational and request, removing previous "validators changed"?

Reply:
IMPORTANT: add event and new Gear lib structs to rust Abi